### PR TITLE
feat(client): integrate categories from backend

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#142 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: client/src/apis/blogs.js [hunk 1]
```diff
@@ -54,10 +54,25 @@ const getUserBlogs = async () => {
...
+  try {
+
+    const categories = categoryList.join(',');
+    const response = await axios.get(`/home/${pageNumber}?search=${searchString}&category=${categories}`);
+    return response.data.blogs;
+  } catch(e) {
+    if(!e.response) {
+      throw e;
+    }
+    throw new Error(e.response.data);
```
Comment (nihalanisumit) [inline client/src/apis/blogs.js:68]: need to create error boundaries in frontend since you have been throwing errors.

**Review by nihalanisumit — APPROVED** [review]
LGTM

File: client/src/apis/blogs.js (hunk 1 above)
Comment (piyushagwl007) [reply client/src/apis/blogs.js:68]: Yes handled this scenario

**Comment by piyushagwl007** [comment]
Category API integration

**Pull request merged**